org.kde.kmplayer.json

```
@@ -0,0 +1,372 @@
+{
+  "app-id": "org.kde.kmplayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "kmplayer",
+  "rename-icon": "kmplayer",
+  "rename-desktop-file": "kmplayer.desktop",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
```

```suggestion
    "--socket=x11",
```

It doesn't work on Wayland.